src/javascript/app/pages/trade/tradepage.js

```diff
@@ -26,57 +26,57 @@ const TradePage = (() => {
     };
 
     const init = () => {
-        State.set('is_trading', true);
-        Price.clearFormId();
-        if (events_initialized === 0) {
-            events_initialized = 1;
-            TradingEvents.init();
-        }
-
-        BinarySocket.wait('authorize').then(() => {
-            Header.displayAccountStatus();
-            if (Client.get('is_virtual')) {
-                Header.upgradeMessageVisibility(); // To handle the upgrade buttons visibility
-                // if not loaded by pjax, balance update function calls TopUpVirtualPopup
-                if (State.get('is_loaded_by_pjax')) {
-                    BinarySocket.wait('balance').then(() => {
-                        TopUpVirtualPopup.init(State.getResponse('balance.balance'));
-                    });
-                }
+        if (!Client.isMFAccount()) {
```

Comment:
please use a return if client is financial instead so the indentation of the below lines don't change